### PR TITLE
Podcast search

### PR DESCRIPTION
**Pull request by kiverso**
Add search endpoint to the microservice, and a new serializer to the podcast results cleaner to serialize search results for delivery back to our rails app.

### CODE REVIEW

File: poro/podcast_results_cleaner.rb [hunk 1]
```diff
@@ -11,6 +11,29 @@ def get_podcast(podcast_id)
...
+              }
+      }
+  end
+
+  def format_search_results(search_results)
+    search_results.map do |result|
+      {
+        title: result[:title_original],
+        description: result[:description_original],
+        thumbnail: result[:thumbnail],
```
Comment (janegreene) [inline poro/podcast_results_cleaner.rb:30]: I'm excited about the prospect of a thumbnail.

File: app.rb [hunk 2]
```diff
@@ -13,4 +13,10 @@ class PodcastMicroService < Sinatra::Base
     podcast_cleaner = PodcastResultsCleaner.new
     JSON podcast_cleaner.get_podcast(params[:podcast_id])
   end
+
+  get '/search' do
```
Comment (muydanny) [inline app.rb:17]: solid!

File: services/podcast_results_service.rb [hunk 3]
```diff
@@ -6,6 +6,9 @@
 
 class PodcastResultsService
 
+  def search_results(search_param)
+    get_json("/search?q=#{search_param}&type=podcasts&only_in=title")
```
Comment (muydanny) [inline services/podcast_results_service.rb:10]: This is awesome. Great job thinking to only search the term within the title.

Comment (janegreene) [inline services/podcast_results_service.rb:10]: Scopes to search in title and only title.

**Pull request merged**